kubevirtci

```diff
@@ -2,20 +2,20 @@
 
 set -e
 
-export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-k8s-1.23}
-export TENANT_CLUSTER_KUBERNETES_VERSION=${TENANT_CLUSTER_KUBERNETES_VERSION:-v1.24.9}
-export KUBEVIRTCI_TAG=${KUBEVIRTCI_TAG:-2205231118-f12b50e}
+export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-k8s-1.27}
+export TENANT_CLUSTER_KUBERNETES_VERSION=${TENANT_CLUSTER_KUBERNETES_VERSION:-v1.27.6}
```

Comment:
i don't think we have a ubuntu image for v1.27 yet. The highest image i see is `quay.io/capk/ubuntu-2004-container-disk:v1.26.0`